### ISSUE TITLE
Job Garages

### ISSUE THREAD

**Issue by NoLoveOrange** [LukeWasTakenn/luke_garages#103]
Hi luke sorry to bother you I were using your script for the past weeks and I would like to implement job garages in my server so that police and other societys can take their cars how can I do it?

**Comment by LukeWasTakenn** [comment]
Set the job for the garage using the `job` property and make sure the `owner` for the vehicle is set to the job name.

**Comment by NoLoveOrange** [comment]
Now my problem is that when one car goest to the impound it doesnt show up until I spawn other car in the polyzone, other issue is that I always have do restart your script by the console to work can you give me a hand?
![Captura de Ecrã (835)](https://user-images.githubusercontent.com/108767841/178124092-4a2bf102-5c70-49e0-867c-399b70034d61.png)
![Captura de Ecrã (836)](https://user-images.githubusercontent.com/108767841/178124094-92e9a20c-9bdb-47d4-812e-4c96b655854a.png)

**Comment by LukeWasTakenn** [comment]
> Now my problem is that when one car goest to the impound it doesnt show up until I spawn other car in the polyzone,

Some people reported this, I was not able to reproduce it at all.

> other issue is that I always have do restart your script by the console to work can you give me a hand?

Wrong resource start other if I had to guess, start qtarget before starting my garage resoruce.

**Comment by NoLoveOrange** [comment]
I realize a thing today that is that impound just does not work if there are not any other cars in the entire map because if there is at leat one it works

**Comment by LukeWasTakenn** [comment]
Duplicate of #51
